dotcom-rendering/src/web/experiments/tests/shady-pie-click-through.ts

```diff
@@ -0,0 +1,22 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const shadyPieClickThrough: ABTest = {
+	id: 'shadyPieClickThrough',
+	start: '2022-09-26',
+	expiry: '2022-10-11',
+	author: 'Emma Imber',
+	description:
+		'Test the click through rate of the new labs shady pie component',
+	audience: 2 / 100,
+	audienceOffset: 0,
+	audienceCriteria: 'All pageviews',
+	successMeasure:
+		'Getting a statistically significant measure of the click through rate',
+	canRun: () => true,
+	variants: [
+		{
+			id: 'variant',
+			test: (): void => {},
+		},
```

Comment:
Historically we've included an additional control group as one of these variants. This is to ensure we collect measurements for a group without the `variant` logic enabled. Do we need one for this test here?

```
		{
			id: 'variant',
			test: (): void => {},
		},
		{
			id: 'control,
			test: (): void => {},
		},
```

Reply:
We don't need one for this test as our control is effectively the users with an ad blocker

Reply:
Thanks for clarifying!